tests/backup/backup_helper.go

```diff
@@ -4376,3 +4376,51 @@ func UpdateCluster(clusterName string, clusterUid string, kubeConfigPath string,
 	}
 	return status, err
 }
+
+// DeleteAllBackups deletes all backup from the given context and org
+func DeleteAllBackups(ctx context.Context, orgId string) error {
+	bkpEnumerateReq := &api.BackupEnumerateRequest{
+		OrgId: orgId,
+	}
+	curBackups, err := Inst().Backup.EnumerateBackup(ctx, bkpEnumerateReq)
+	if err != nil {
+		return err
+	}
+	errChan := make(chan error, len(curBackups.GetBackups()))
+	semaphore := make(chan int, 4)
+	var wg sync.WaitGroup
+	for _, bkp := range curBackups.GetBackups() {
+		wg.Add(1)
+		go func(bkp *api.BackupObject) {
+			semaphore <- 0
+			defer wg.Done()
+			defer func() { <-semaphore }()
+			bkpDeleteRequest := &api.BackupDeleteRequest{
+				Name:    bkp.GetName(),
+				OrgId:   bkp.GetOrgId(),
+				Uid:     bkp.GetUid(),
+				Cluster: bkp.GetCluster(),
```

Comment:
with 2.6.0 it become optional to have cluster ref, delete request will internally fetch the reference.

Reply:
@ak-px 
That means mentioning "Cluster: bkp.GetCluster(),"should also be ok?

Reply:
No need to pass cluster Ref, delete request will fetch internally the clusterRef from backupInfo

Reply:
done